source/index.js

```diff
@@ -13,12 +14,43 @@ renderer.roundPixels = true
 document.getElementById("mount").appendChild(renderer.view)
 window.game = new Pixi.Container()
 game.addChild(new Junkership())
+game.score=0
 
 game.addChild(new Trashbot(Reference.GAME_WIDTH, Reference.GAME_HEIGHT/2))
 
 var loop = new Afloop(function(delta) {
     game.children.forEach((child) => {
         child.update(delta)
+
+        if (child instanceof Trashbot ||
+            child instanceof Junkership){
+            collisionCheck(child)
+        }
     })
     renderer.render(game)
+    console.log("Score: " + game.score)
 })
+
+function collisionCheck(child){
+    var x1 = child.position.x
+    var y1 = child.position.y
+    var w1 = child.width
+    var h1 = child.height
+
+    var x2, y2, w2, h2
+
+    game.children.forEach((toCompare) => {
```

Comment:
Until we start seeing slowdown in the game, we have no need to optimize, but just keep in mind, this is a O(n^2), because (in the worst case) we're checking every child against every other child.